src/main/resources/lang/fr_fr.json

```diff
@@ -38,17 +38,24 @@
 	"menu.view.languages.summary": "La nouvelle langue sera appliquée lors du prochain redémarrage.",
 	"menu.view.languages.ok": "Ok",
 	"menu.view.search": "Rechercher",
+	"menu.collab": "Collab",
+	"menu.collab.connect": "Se connecter à un serveur",
+	"menu.collab.connect.error": "Erreur lors de la connexion au serveur",
+	"menu.collab.disconnect": "Se déconnecter",
+	"menu.collab.server.start": "Démarrer le serveur",
+	"menu.collab.server.start.error": "Erreur lors du démarrage du serveur",
+	"menu.collab.server.stop": "Arrêter le serveur",
 	"menu.help": "Aide",
 	"menu.help.about": "À propos",
 	"menu.help.about.title": "%s - À propos",
 	"menu.help.about.ok": "Ok",
 	"menu.help.github": "Page Github",
 
 	"popup_menu.rename": "Renommer",
-	"popup_menu.javadoc": "Éditer le Javadoc",
+	"popup_menu.javadoc": "Éditer les javadocs",
 	"popup_menu.inheritance": "Afficher l'héritage",
 	"popup_menu.implementations": "Afficher les implémentations",
-	"popup_menu.calls": "Afficher les appels",
+	"popup_menu.calls": "Afficher les appels (tous)",
```

Comment:
I would PR these changes to the Fabric Enigma repo, remember everything you put here will end up in the multiplayer PR.

Reply:
Okay I'll rename those in a separate pr